cert-manager/go.mod

```
@@ -0,0 +1,84 @@
+module github.com/keptn/lifecycle-toolkit/cert-manager
+
+go 1.19
+
+require (
+	github.com/go-logr/logr v1.2.3
+	github.com/pkg/errors v0.9.1
+	github.com/stretchr/testify v1.8.1
+	k8s.io/api v0.25.4
+	k8s.io/apiextensions-apiserver v0.25.0
```

```suggestion
	k8s.io/apiextensions-apiserver v0.25.4
```

let's use same version everywhere

---

have you tried v0.25.5 ? It's the newest patch version, maybe worth updating

---

will do in the next pr since here is unused